### PR TITLE
Add cf-component-header and cf-component-logo

### PR DESCRIPTION
**Pull request by tajo**
screen shot 2017-08-28 at 5 09 53 pm

**Demo of responsivity and stickiness:**
header

### CODE REVIEW

**Comment by neynah** [comment]
🎉

**Comment by hturan** [comment]
Looking good! I think it would make sense to separate some of these out though. If I want a `Logo` or `Header`, looking inside `cf-component-page` seems unintuitive to me.

Would you be against making a `cf-component-logo` and `cf-component-header`?

**Comment by tajo** [comment]
Hamburger added

screen shot 2017-08-28 at 5 38 32 pm

**Comment by tajo** [comment]
@hturan Yea, I should have done it. I'm also working on footer, so `cf-component-footer` makes sense as well.

**Comment by tajo** [comment]
@hturan split

**Comment by hturan** [comment]
The logo SVGs are pretty messy. It's fine for now, but I know that @mrmrs has been working on a `Logo` primitive component (with `light` and `inline` props for the different variants), might be worth syncing up 👍

**Comment by tajo** [comment]
@hturan Yea, it would be really nice to have one React component `Logo.js` with props as `light`, `noText`, `inline`, `blackAndWhite`

File: packages/cf-component-header/example/basic/component.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { Box } from 'cf-component-box';
+import { Header, NavList, NavItem, Hamburger } from 'cf-component-header';
+import { Logo } from 'cf-component-logo';
+
+const ViewComponent = () => (
+  <Header>
+    <Box display="flex">
+      <Hamburger />
```
Comment (jculvey) [inline packages/cf-component-header/example/basic/component.js]: 🍔🍔🍔🍔🍔🍔🍔🍔🍔🍔🍔🍔🍔🍔🍔

**Pull request merged**